exe/fil-tools/cmd/deals_retrieve.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"io"
+	"os"
+	"path"
+
+	"github.com/caarlos0/spin"
+	"github.com/ipfs/go-cid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	retrieveCmd.Flags().StringP("address", "a", "", "wallet address of the file to fetch")
```

Comment:
Nit: maybe confusing descripton? Alternative: `wallet address to use funds for retrieval`.

Reply:
cool, updated.